gateway/src/main/java/io/camunda/zeebe/gateway/RequestMapper.java

```diff
@@ -52,13 +53,16 @@
 import io.camunda.zeebe.protocol.impl.encoding.MsgPackConverter;
 import io.camunda.zeebe.protocol.impl.stream.job.JobActivationProperties;
 import io.camunda.zeebe.protocol.impl.stream.job.JobActivationPropertiesImpl;
+import java.util.regex.Pattern;
 import org.agrona.DirectBuffer;
 import org.agrona.concurrent.UnsafeBuffer;
+import org.springframework.util.StringUtils;
```

Comment:
❌ We try to limit depending on Spring. Zeebe only uses Spring as an easy application container, including configuration loading and some actuator endpoints. If you want string-specific util methods, I recommend using the commons libraries or Java's standard library instead.